svelte/demo/src/bootstrap/samples/slots/Usage.route.svelte

```diff
@@ -4,6 +4,4 @@
 
 <Alert type="primary" dismissible={false}>Label provided by slot</Alert>
 <Alert type="secondary" dismissible={false} children="Label provided by property" />
-<Alert type="info" dismissible={false} children="When both prop and slot are provided, the prop's content will take precedence."
-	>This content is ignored.</Alert
->
+<p>Trying to set children through snippet and property at the same time will result in an error.</p>
```

Comment:
Since we have not released 1.0 yet, I would say we do not need migration guidelines yet.

Reply:
True, but do we need to explicitly say that paragraph? is it to avoid opening issues about it ? :)

Reply:
That paragraph hopes to explicitly describe the behavior, in case people switch framework while staying on this page, since the behavior is different between Angular, React and Svelte.